assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -99,6 +102,12 @@ angular.module("openshiftConsole")
             });
         }
 
+        var watches = [];
+        watches.push(DataService.watch("limitranges", context, function(limitRanges) {
```

Comment:
limit ranges very very rarely change, wondering if we should just list here and save a websocket connection,  when we originally talked about watching for this stuff it was for quota which was going to be updating potentially.